### PR TITLE
Update the contribute guide

### PR DESCRIPTION
**Pull request by alabulei1**
## Explanation

* Update Contribute.md following CNCF TAG contributor guideline
* Remove the location of LLM workload on GPU
* Add an explanation about the relationship between WasmEdge and Second State

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/contribute/contribute.md [hunk 1]
```diff
@@ -2,72 +2,166 @@
...
 Please submit a pull request broken down into small changes bit by bit. A pull request consisting of many features and code changes may take a lot of work to review. It is recommended to submit pull requests incrementally.
 
-<!-- prettier-ignore -->
-:::note
-If you split your pull request into small changes, please ensure any changes that go to the main branch will not break anything. Otherwise, it can only be merged once this feature is complete.
-:::
+Generally, once your pull request has been opened, it will be assigned to one or more reviewers. Those reviewers will do a thorough code review, looking for correctness, bugs, opportunities for improvement, documentation and comments, and coding style. If your PR is not ready to review, please mark you PR as draft.
 
-### Fork and Clone the Repository
+The reviewers will give you some feedback in three workdays.
```
Comment (hydai) [inline docs/contribute/contribute.md:102]: Why do we choose these time periods? Do we have any reasons?

Comment (alabulei1) [reply docs/contribute/contribute.md:102]: It's recommended by the CNCF tag contributor strategy.

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**